engine_json_features.ipynb

```diff
@@ -246,9 +246,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 4.2. Generative React\n",
+    "### Part 2: Generative React\n",
     "\n",
-    "We can also use a generative react to generate any number of more or less surprising inventories.\n",
+    "We can also use a generative `react` (`react` is generative when `desired_conviction` is specified.) to generate any number of more or less surprising inventories.\n",
```

Comment:
Suggestion: could make a nice link to the correct part of the documentation for generative/descriminative!